.github/workflows/publish-linux.yml

```diff
@@ -57,13 +57,13 @@ jobs:
         working-directory: noir
         run: |
           cargo install cross --force --git https://github.com/cross-rs/cross
-          cross build --package nargo --release --target=${{ matrix.target }} --no-default-features --features plonk_bn254_wasm
+          cross install --release --target=${{ matrix.target }} --no-default-features --features plonk_bn254_wasm

```

Comment:
I forgot this is the `cross` tool and not cargo directly. It doesn't seem like cross has an install command

Reply:
Ah, whoops I missed that. Both 5 letters beginning with c

Reply:
Probably sensible to just manually copy the built binary into `~/.cargo/bin`

Reply:
I was searching the source code and I think they accept a `--target-dir` flag that we could use

Reply:
They also might pass through the install command, I'm turned around in their code.

I think we need to remove all the `--release` flags since that's the default for install and see what happens.